modules/account-lib/src/coin/near/transaction.ts

```diff
@@ -94,7 +94,8 @@ export class Transaction extends BaseTransaction {
    *
    * @param rawTransaction
    */
-  fromRawTransaction(rawTransaction: string): void {
+  fromRawTransaction(rawTx: string): void {
+    const rawTransaction = utils.base58Encode(Buffer.from(rawTx, 'base64'));
```

Comment:
can we eliminate base58encode by combine with  base_decode below?

Reply:
Done